e2e/test_module.py

```diff
@@ -154,6 +166,63 @@ def _run_test(test_dict: Dict, db: Memgraph):
             db.execute(test_query)
 
 
+def _get_nodes_and_relationships(
+    nodes_query: str, relationships_query: str, db: Memgraph
+):
+    return (
+        list(db.execute_and_fetch(nodes_query)),
+        list(db.execute_and_fetch(relationships_query)),
+    )
+
+
+def _test_export(test_dir: Path, db: Memgraph):
+    """
+    Testing export modules.
+    """
+
+    input_dict = _load_yaml(test_dir.joinpath(TestConstants.INPUT_FILE))
+
+    queries = input_dict.get(TestConstants.EXPORT_TEST_E2E_INPUT_QUERIES, None)
+    db.execute(queries)
+
+    nodes_query = input_dict.get(TestConstants.EXPORT_TEST_E2E_NODES, None)
+    relationships_query = input_dict.get(
+        TestConstants.EXPORT_TEST_E2E_RELATIONSHIPS, None
+    )
+
+    old_nodes, old_relationships = _get_nodes_and_relationships(
+        nodes_query, relationships_query, db
+    )
+
+    test_dict = _load_yaml(test_dir.joinpath(TestConstants.TEST_FILE))
+    export_query = test_dict[TestConstants.EXPORT_TEST_E2E_EXPORT_QUERY].replace(
+        TestConstants.EXPORT_TEST_E2E_PLACEHOLDER_FILENAME,
+        "".join(["'", TestConstants.EXPORT_TEST_E2E_OUTPUT_FILE, "'"]),
+    )
+    import_query = test_dict[TestConstants.EXPORT_TEST_E2E_IMPORT_QUERY].replace(
+        TestConstants.EXPORT_TEST_E2E_PLACEHOLDER_FILENAME,
+        "".join(["'", TestConstants.EXPORT_TEST_E2E_OUTPUT_FILE, "'"]),
+    )
+
+    db.execute(export_query)
```

Comment:
As we talked on Slack, I think we don't need export part here, so we can remove it